implementations/rust/ockam/ockam/src/stream/mod.rs

```diff
@@ -14,7 +14,9 @@ use crate::{
 use core::{ops::Deref, time::Duration};
 use ockam_core::compat::rand::{self, Rng};
 use ockam_core::compat::string::String;
-use ockam_core::{Decodable, RouteBuilder};
+use ockam_core::{Decodable, RouteBuilder, TransportType};
+
+pub const STREAM: TransportType = TransportType::new(16);
```

Comment:
Not sure if this is the name people would want. Please let me know if something else is desired.

Reply:
@thomcc I don't know what was the initial intention of @spacekookie here, but I'm pretty sure this PR haven't changed how this code works, so should be good to merge